### ISSUE TITLE
Crash when opening world: creeper model not found

### ISSUE THREAD

**Issue by iconmaster5326** [bernie-g/geckolib#99]
When I load a world, the game crashes, claiming it cannot find a model for a creeper:

[debug.log](https://github.com/bernie-g/geckolib/files/5912763/debug.log)

I'm using GeckoLib 3.0.10 on MC 1.16.5. This is occurring while I'm developing my own mod, but it seems my mod has nothing to do with this crash.

Here's a relevant-seeming snippet from the log:

```
[11:34:15] [Render thread/INFO] [STDOUT/]: [net.minecraft.util.registry.Bootstrap:printToSYSOUT:123]: ---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 2/2/21, 11:34 AM
Description: Rendering entity in world

software.bernie.geckolib3.geo.exception.GeoModelException: geckolib3:geo/creeper.geo.json: Could not find model. If you are getting this with a built mod, please just restart your game.
        at software.bernie.geckolib3.model.AnimatedGeoModel.getModel(AnimatedGeoModel.java:110) ~[geckolib-forge-1.16.5-3.0.10_mapped_snapshot_20201028-1.16.3.jar:3.0.10] {re:classloading}
        at software.bernie.geckolib3.renderers.geo.GeoReplacedEntityRenderer.render(GeoReplacedEntityRenderer.java:159) ~[geckolib-forge-1.16.5-3.0.10_mapped_snapshot_20201028-1.16.3.jar:3.0.10] {re:classloading}
        at software.bernie.geckolib3.renderers.geo.GeoReplacedEntityRenderer.render(GeoReplacedEntityRenderer.java:78) ~[geckolib-forge-1.16.5-3.0.10_mapped_snapshot_20201028-1.16.3.jar:3.0.10] {re:classloading}
        at net.minecraft.client.renderer.entity.EntityRendererManager.renderEntityStatic(EntityRendererManager.java:251) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
        at net.minecraft.client.renderer.WorldRenderer.renderEntity(WorldRenderer.java:1202) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:runtimedistcleaner:A}
        at net.minecraft.client.renderer.WorldRenderer.updateCameraAndRender(WorldRenderer.java:1010) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:runtimedistcleaner:A} 
        at net.minecraft.client.renderer.GameRenderer.renderWorld(GameRenderer.java:607) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
        at net.minecraft.client.renderer.GameRenderer.updateCameraAndRender(GameRenderer.java:425) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
        at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:976) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}      
        at net.minecraft.client.Minecraft.run(Minecraft.java:607) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
        at net.minecraft.client.main.Main.main(Main.java:184) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {re:classloading,pl:runtimedistcleaner:A}
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64) ~[?:?] {}
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
        at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?] {}
        at net.minecraftforge.userdev.FMLUserdevClientLaunchProvider.lambda$launchService$0(FMLUserdevClientLaunchProvider.java:52) ~[forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.9.jar:?] {}
        at net.minecraftforge.userdev.LaunchTesting.main(LaunchTesting.java:105) [forge-1.16.5-36.0.13_mapped_snapshot_20201028-1.16.3.jar:?] {}
```

**Comment by bernie-g** [comment]
@AzureDoom

**Comment by AzureDoom** [comment]
Dupe of #81 but should be fixed in the newest version (v3.0.12). Please comment to the other issue if it happens again.
